source/tests/pd/test_change_bias.py

```
@@ -0,0 +1,150 @@
+        wrapper.set_state_dict(state_dict["model"])
+        updated_bias = wrapper.model["Default"].get_out_bias()
+        expected_model = model_change_out_bias(
+            self.trainer.wrapper.model["Default"],
+            self.sampled,
+            _bias_adjust_mode="change-by-statistic",
+        )
+        expected_bias = expected_model.get_out_bias()
+        np.testing.assert_allclose(updated_bias.numpy(), expected_bias.numpy())
+
+    def test_change_bias_with_data_sys_file(self):
+        tmp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".txt")
```

Ruff

Use a context manager for opening files

(SIM115)